### PR TITLE
Document some missing parameters and outputs

### PR DESCRIPTION
**Pull request by erasche**

### CODE REVIEW

File: bioblend/galaxy/datatypes/__init__.py [hunk 1]
```diff
@@ -14,6 +14,12 @@ def get_datatypes(self, extension_only=False, upload_only=False):
         """
         Get the list of all installed datatypes.
 
+        :type extension_only: bool
+        :param extension_only: Return only the extension rather than the datatype name
+
+        :type upload_only: bool
+        :param upload_only: Filter for datatypes wihch can be uploaded
```
Comment (nsoranzo) [inline bioblend/galaxy/datatypes/__init__.py]:
```suggestion
        :param upload_only: Whether to return only datatypes which can be uploaded
```

File: bioblend/galaxy/forms/__init__.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def create_form(self, form_xml_text):
         :param  form_xml_text: Form xml to create a form on galaxy instance
 
         :rtype:     str
-        :returns:   Unique url of newly created form with encoded id
+        :return:   Unique url of newly created form with encoded id
```
Comment (nsoranzo) [inline bioblend/galaxy/forms/__init__.py]:
```suggestion
        :return:   Unique URL of newly created form with encoded id
```

File: bioblend/galaxy/ftpfiles/__init__.py [hunk 3]
```diff
@@ -14,6 +14,9 @@ def get_ftp_files(self, deleted=False):
         """
         Get a list of local files.
 
+        :type deleted: bool
+        :param deleted: Include deleted files
```
Comment (nsoranzo) [inline bioblend/galaxy/ftpfiles/__init__.py]:
```suggestion
        :param deleted: Whether to include deleted files
```

File: bioblend/galaxy/users/__init__.py [hunk 4]
```diff
@@ -17,6 +17,10 @@ def get_users(self, deleted=False, f_email=None, f_name=None, f_any=None):
         Get a list of all registered users. If ``deleted`` is set to ``True``,
         get a list of deleted users.
 
+        :type deleted: bool
+        :param deleted: Include deleted users
```
Comment (nsoranzo) [inline bioblend/galaxy/users/__init__.py]:
```suggestion
        :param deleted: Whether to include deleted users
```

File: bioblend/galaxy/tools/__init__.py [hunk 5]
```diff
@@ -90,6 +93,9 @@ def show_tool(self, tool_id, io_details=False, link_details=False):
 
         :type link_details: bool
         :param link_details: whether to get also link details
+
+        :rtype: dict
+        :return: Information about a tool's interface
```
Comment (nsoranzo) [inline bioblend/galaxy/tools/__init__.py]:
```suggestion
        :return: Information about the tool's interface
```

File: bioblend/galaxy/histories/__init__.py [hunk 6]
```diff
@@ -52,6 +52,9 @@ def get_histories(self, history_id=None, name=None, deleted=False):
         :type name: str
         :param name: Name of history to filter on
 
+        :type deleted: bool
+        :param deleted: Include deleted histories
```
Comment (nsoranzo) [inline bioblend/galaxy/histories/__init__.py]:
```suggestion
        :param deleted: whether to filter for the deleted histories (``True``)
          or for the non-deleted ones (``False``)
```

File: bioblend/galaxy/histories/__init__.py [hunk 7]
```diff
@@ -442,6 +493,12 @@ def delete_history(self, history_id, purge=False):
         :type purge: bool
         :param purge: if ``True``, also purge (permanently delete) the history
 
+        :rtype: dict
+        :return: An error object if an error occurred or a dictionary
+                 containing; id: the encoded id of the history, deleted: if the
```
Comment (nsoranzo) [inline bioblend/galaxy/histories/__init__.py]:
```suggestion
                 containing: ``id`` (the encoded id of the history), ``deleted`` (if the
```

File: bioblend/galaxy/histories/__init__.py [hunk 8]
```diff
@@ -442,6 +493,12 @@ def delete_history(self, history_id, purge=False):
...
+                 history was marked as deleted, purged: if the history was
```
Comment (nsoranzo) [inline bioblend/galaxy/histories/__init__.py]:
```suggestion
                 history was marked as deleted), ``purged`` (if the history was
```

File: bioblend/galaxy/histories/__init__.py [hunk 9]
```diff
@@ -442,6 +493,12 @@ def delete_history(self, history_id, purge=False):
...
+                 purged
```
Comment (nsoranzo) [inline bioblend/galaxy/histories/__init__.py]:
```suggestion
                 purged).
```

**Comment by erasche** [comment]
Thanks for the feedback @nsoranzo!

**Comment by nsoranzo** [comment]
Sorry, it's conflicting with the merge of #283.
Do you have time to rebase?

**Pull request merged**